code/game/machinery/computer/medical.dm

```diff
@@ -96,7 +96,13 @@
 					var/list/records = list()
 					data["records"] = records
 					for(var/datum/data/record/R in sortRecord(GLOB.data_core.general))
-						records[++records.len] = list("ref" = "\ref[R]", "id" = R.fields["id"], "name" = R.fields["name"])
+						records[++records.len] = list(
```

Comment:
```suggestion
						records += list(
```
this should have the same functionality but you should check

Reply:
Doing this breaks filtering for some reason

Reply:
ah, it may be because it needs to be a double list in this case due to `+=`, optional if you want to clean up this code

Reply:
Its the way which the rest of the file does it, which albeit is bad, but it'll be consistent